cli/src/agent.gql

```diff
@@ -0,0 +1,71 @@
+query AgentStatus {
+    agentStatus {
+        isInitialized
+        isUnlocked
+        did
+        didDocument
+        error
+    }
+}
+
+mutation RequestCapability($appName: String!, $appDesc: String!, $appUrl: String!, $capabilities: String!) {
+    agentRequestCapability(appName: $appName, appDesc: $appDesc, appUrl: $appUrl, capabilities: $capabilities)
+}
```

Comment:
are we able to have these queries generated / imported from core in anyway?

Reply:
They are currently inline in the JS client code. And they are a bit different here (the name) since it has to match with the Rust struct name in order to use clap's derivation style... so I would keep it this way which is how GraphQL is meant: the client is in charge of the query...